src/patternfly/components/Form/form.scss

```diff
@@ -167,6 +167,7 @@
   display: flex;
   flex-wrap: wrap;
   margin-bottom: var(--pf-c-form__actions--MarginBottom);
```

Comment:
Looking at this more, I don't know if form__actions should have a margin? It seems so specific to one example and maybe grid can handle the layout.

Reply:
It just needs to offset the bottom margin of the buttons, and the margin on the buttons is for when the buttons wrap, so they have space between them. We don't want to use grid because we don't want columns. What we need is `row-gap` with `display: flex`, which [should be coming soonish](https://github.com/w3c/csswg-drafts/issues/1696). Until then, this is the poor man's form of a `row-gap`.

Reply:
Ok no worries! It would be awesome if there was a selector for overflown-content